libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -218,119 +218,27 @@ bool AC_AutoTune::init_position_controller(void)
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: Paused: Pilot Override Active");
-        return;
-    }
-    switch (step) {
-    case WAITING_FOR_LEVEL:
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: Leveling (%s %4.1f > %4.1f)", level_issue_string(), (double)(level_problem.current*0.01f), (double)(level_problem.maximum*0.01f));
-        return;
-    case UPDATE_GAINS:
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: Updating Gains");
-        return;
-    case TWITCHING:
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: Twitching");
-        return;
-    }
-    gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: unknown step");
-}
-
-const char *AC_AutoTune::type_string() const
-{
-    switch (tune_type) {
-    case RD_UP:
-        return "Rate D Up";
-    case RD_DOWN:
-        return "Rate D Down";
-    case RP_UP:
-        return "Rate P Up";
-    case SP_DOWN:
-        return "Angle P Down";
-    case SP_UP:
-        return "Angle P Up";
-    }
-    return "Bug";
-}
-
 void AC_AutoTune::do_gcs_announcements()
 {
     const uint32_t now = AP_HAL::millis();
     if (now - announce_time < AUTOTUNE_ANNOUNCE_INTERVAL_MS) {
         return;
     }
-    float tune_rp = 0.0f;
-    float tune_rd = 0.0f;
-    float tune_sp = 0.0f;
-    float tune_accel = 0.0f;
+
     char axis_char = '?';
     switch (axis) {
     case ROLL:
-        tune_rp = tune_roll_rp;
-        tune_rd = tune_roll_rd;
-        tune_sp = tune_roll_sp;
-        tune_accel = tune_roll_accel;
         axis_char = 'R';
         break;
     case PITCH:
-        tune_rp = tune_pitch_rp;
-        tune_rd = tune_pitch_rd;
-        tune_sp = tune_pitch_sp;
-        tune_accel = tune_pitch_accel;
         axis_char = 'P';
         break;
     case YAW:
-        tune_rp = tune_yaw_rp;
-        tune_rd = tune_yaw_rLPF;
-        tune_sp = tune_yaw_sp;
-        tune_accel = tune_yaw_accel;
         axis_char = 'Y';
         break;
     }
 
-    gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: (%c) %s", axis_char, type_string());
-    send_step_string();
-    if (!is_zero(lean_angle)) {
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: lean=%f target=%f", (double)lean_angle, (double)target_angle);
-    }
-    if (!is_zero(rotation_rate)) {
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: rotation=%f target=%f", (double)(rotation_rate*0.01f), (double)(target_rate*0.01f));
-    }
-    switch (tune_type) {
-    case RD_UP:
-    case RD_DOWN:
-    case RP_UP:
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: p=%f d=%f", (double)tune_rp, (double)tune_rd);
-        break;
-    case SP_DOWN:
-    case SP_UP:
-        gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: p=%f accel=%f", (double)tune_sp, (double)tune_accel);
-        break;
-    }
-    gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: success %u/%u", counter, AUTOTUNE_SUCCESS_COUNT);
+    gcs().send_text(MAV_SEVERITY_INFO, "AutoTune: (%c) %u/%u", axis_char,  counter, AUTOTUNE_SUCCESS_COUNT);
```

Comment:
this is less clear. 
IHMO we should just have a counter to the global axis with want to tune to follow the progression. There it is just a bunch of letter and number without meaning for most user